### ISSUE TITLE
Simplest/first version of dashboard -> subset of issue 57

### ISSUE THREAD

**Issue by mhelali** [pickleworks/WhenWhereScheduler#80]
Original entry:  https://github.com/pickleworks/WhenWhereScheduler/issues/57

The most simple Dashboard logic:

Same as normal calendar page that provider sees with this additional logic (reminder that now there is just the one SAME page that is the dashboard for a PERSON. A person is usually a "client" but they can also be "provider". The current calendar page is for "provider". The logic will be something like:

After the PERSON logs in successfully:

a- open the dashboard for PERSON

if PERSON is also a PROVIDER:

b- add provider features (such as "New Event") -- what we currently have.

Here is minimal functionality we need for PERSON dashboard:

1- select a provider (minimally be able to select ONE -- later would like to be able to select multiple as shown in picture of issue 57).

2- if no provider have the ability to be connected to one (in full functionality this will be done in several way, most common is by clicking on the invite email from provider, and also being able to invite/ask to be a client of a provider by entering their email. In this version do whatever is the easiest/quickest (code wise) to connect to a provider.

3- Show the available slots of that provider. Be able to join. (full functionality will be also entering other data (like what to bring, etc). But for now just be able to join (and ideally un-join).

Basically the iphone app will be doing this same functionality. But I wont' be able to make an Android version so we need a web version.

Originally we had talked about the simplicity of having just one dashboard page. But if code wise it is simpler to have the provider dash be separate than client dash, that is okay. The iphone app is going to be purely for client.

**Comment by mhelali** [comment]
So obviously the search options of the full dashboard (as stated in https://github.com/pickleworks/WhenWhereScheduler/issues/57) don't need to be implemented in this first pass.
